verilog/analysis/verilog_project.cc

```diff
@@ -195,11 +197,22 @@ absl::StatusOr<VerilogSourceFile*> VerilogProject::OpenIncludedFile(
   return inserted.first->second->Status();
 }
 
-void VerilogProject::AddVirtualFile(absl::string_view referenced_filename,
+void VerilogProject::AddVirtualFile(absl::string_view resolved_filename,
                                     absl::string_view content) {
+  std::string referenced_filename(resolved_filename);
+  // Make the virtual file relative to the include directory.
+  for (absl::string_view include_path : include_paths_) {
+    if (absl::StartsWith(resolved_filename, include_path)) {
+      referenced_filename = std::string(absl::StripPrefix(
+          resolved_filename, absl::StrCat(include_path, "/")));
```

Comment:
What happens if `include_path` already has a slash at the end ?

Reply:
This is obsolete.